### PR TITLE
My adventure

### PR DESCRIPTION
**Pull request by coder-neo**
Please review this, @casey-collab

### CODE REVIEW

File: english/naruto/seventh-hokage.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Naruto Uzumaki (うずまきナルト, Uzumaki Naruto) is a shinobi of
```
Comment (casey-collab) [inline english/naruto/seventh-hokage.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by coder-neo** [comment]
Added reference to my story. Please review again, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :gem:

**Pull request merged**